### PR TITLE
Jlc/feedback and rating models

### PR DESCRIPTION
**Pull request by johanv26**
## Description

Add models and admin for rating and feedback features.

Useful information to include:
- Added for units and courses.
- added public field for feedback and rating according to jira.
image

## Testing instructions
go to your local environment and migrate this branch.
Screenshot from 2023-05-25 17-53-55

### After
Test the model behavior in Admin.
Peek 2023-05-26 12-15

## Additional information

Include anything else that will help reviewers and consumers understand the change.
- For feedback, I select a limit of 300 =).
image

#### ADR :
https://docs.google.com/document/d/1v6nq3pz9hj4RKTildCPbijlrj_TOwyfGeW4cFq_7xbE/edit
### JIRA story
https://edunext.atlassian.net/jira/software/c/projects/FUTUREX/issues/FUTUREX-408
## Checklist for Merge

- [ ] Tested in a remote environment
- [ ] Updated documentation
- [ ] Rebased master/main
- [ ] Squashed commits

### CODE REVIEW

File: eox_nelp/course_experience/models.py [hunk 1]
```diff
@@ -64,6 +64,52 @@ class Meta:
...
+    RATING_OPTIONS = [
+        (0, '0'),
+        (1, '1'),
+        (2, '2'),
+        (3, '3'),
+        (4, '4'),
+        (5, '5')
+    ]
+    author = models.ForeignKey(User, null=True, on_delete=models.SET_NULL)
+    rating = models.IntegerField(null=True, blank=True, choices=RATING_OPTIONS)
```
Comment (andrey-canon) [inline eox_nelp/course_experience/models.py:85]:
```suggestion
    rating_content = models.IntegerField(null=True, blank=True, choices=RATING_OPTIONS)
    rating_instructors = models.IntegerField(null=True, blank=True, choices=RATING_OPTIONS)
```

File: eox_nelp/course_experience/models.py [hunk 2]
```diff
@@ -64,6 +64,52 @@ class Meta:
...
+    course_id = models.ForeignKey(CourseOverview, null=True, on_delete=models.SET_NULL)
+    public = models.BooleanField(null=True, default=True)
```
Comment (andrey-canon) [inline eox_nelp/course_experience/models.py:87]:
```suggestion
    is_public = models.BooleanField(null=True, default=True)
```

**Comment by johanv26** [comment]
@andrey-canon what do you think of the new fields:

**Review by andrey-canon — APPROVED** [review]
I like this but could you add starts to the admin panel instead of selecting a number ?

**Comment by johanv26** [comment]
> I like this but could you add starts to the admin panel instead of selecting a number ?

Starts is for beginning or do you mind stars ⭐in the admin?

**Comment by andrey-canon** [comment]
> > I like this but could you add starts to the admin panel instead of selecting a number ?
> 
> Starts is for beginning or do you mind stars stars in the admin?

Starts in the admin panel, that will look more fancier, it's just a joke

**Pull request merged**